NeuralAmpModeler/projects/NeuralAmpModeler-iOS.xcodeproj/project.pbxproj

```diff
@@ -21,70 +21,76 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
-		4F0D965B23099F5A00BFDED0 /* IPlugProcessor.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 4FFF103420A0E55900D3092F /* IPlugProcessor.cpp */; };
```

Comment:
Can you summarize these changes? Looks like just reorganizing? Or were any added/removed?

Reply:
life is too short! i'm fixing the iOS project. This involves reduplicating the iPlug2OOS template and bringing in the modified .xcodeproj and adding the files, getting it to build

Reply:
Sounds good 😅 Just a dense diff of configgy code that on first scan didn't look like it was doing as much as the red & green implied; figured a quick answer from you would be the fastest way to get this ok'd 🙂